MIGRATION_GUIDE.md

```diff
@@ -4,6 +4,12 @@ This document is meant to help you migrate your Terraform config to the new newe
 describe deprecations or breaking changes and help you to change your configuration to keep the same (or similar) behavior
 across different versions.
 
+## v0.86.0 ➞ v0.87.0
+
+#### *(behavior change)* Schema `data_retention_days`
```

Comment:
I think you have conflicts here (I already merged one change with v0.86.0 ➞ v0.87.0 changes

Reply:
Resolved